_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,41 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Recommended reading:
+  - [#30352: Add PayToAnchor(P2A), OP 1 <0x4e73> as standard output script for spending](https://bitcoincore.reviews/30352)
+
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it.
 
+- PR [30352](https://github.com/bitcoin/bitcoin/pull/30352) added support for PayToAnchor (P2A), a new output type that enables highly efficient keyless anchors (with very low vbyte usage). This new anchor type offers an improvement over previous anchor types (some of which relied on more expensive constructs such as P2WSH).
+
+- The ability of anchors to use amounts below the dust threshold was split off into the ephemeral dust PR we're reviewing today. Ephemeral dust enables additional flexibility and efficiency, for example to enable keyed and keyless anchors to use zero funds from inputs while still enabling a child to bring fees (ala CPFP). This can be useful for L2s (e.g. for the LN).
```

Comment:
Imo these can be removed. The idea for separating them is they're independent features, it might even be confusing to keep talking about them together. Any output type is eligible for ephemeral dust, not just p2a.

Reply:
Agreed. Reduced focus on P2A.